examples/card.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>px-card demo</title>
+
+  <!-- Load webcomponentsjs + Polymer -->
+  <script src="../webcomponentsjs/webcomponents-loader.js"></script>
```

Comment:
Switch out to conditional webcomponents-lite.js loading to keep Polymer 1.x working.